actions/image_partition_action.go

```diff
@@ -192,6 +200,7 @@ type ImagePartitionAction struct {
 	ImageName        string
 	ImageSize        string
 	PartitionType    string
+	PartitionUUID    string
```

Comment:
This should really be called DiskGUID or somesuch (as per the gpt naming) or Disk id/disk signature (as per msdos naming) not partition UUID. 

Partition UUID  typically refers to the per partition UUID  available in GPT tables. Thee naming/descripton actually made me think that was what this was implementing untill i saw the sfdisk callout.